lib/common/tests/options/pcmk__env_option_enabled_test.c

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+#include <crm/common/unittest_internal.h>
+
+#include "mock_private.h"
+
+static void
+disabled_null_value(void **state)
```

Comment:
I wonder if we should mock `pcmk__env_option()`. So far only system calls are mocked. But this test basically tests `pcmk__env_option()` again via `pcmk__env_option_enabled()`, which is redundant.

All we need to test here is whether, **given** a NULL return value from `pcmk__env_option()`, `pcmk__env_option_enabled()` returns false. We don't care what **causes** `pcmk__env_option()` to return NULL.

Reply:
No need to mock pcmk__env_option(), nor repeat all possible tests for it or crm_is_true(). We just want to test that pcmk__env_option_enabled() is combining the two correctly.

Reply:
Cool, so it sounds like I could just pick any given `option` (e.g., `NULL`) that will cause `pcmk__env_option()` to return `NULL`, in order to test this.

Reply:
Done.